content/riak/kv/2.1.1/developing/data-types.md

```diff
@@ -30,7 +30,7 @@ data types: flags, registers, [counters](#counters),
 Of those five types, counters, sets, and maps can be used as
 bucket-level data types, i.e. types that you can interact with directly.
 Flags and registers, however, must be embedded in maps (more on that
-[below](/riak/kv/2.1.1/developing/data-types/#maps).
```

Comment:
In this doc and all other 2.1+ docs:

1. *"Riak KV 2.0+ introduces Riak-specific data types based on convergent replicated data types (CRDTs). While Riak KV was built as a data-agnostic key/value store, Riak Data Types enable you to use Riak KV as a data-aware system and perform transactions on 5 CRDT-inspired data types:"* should read: "Riak KV has Riak-specific data types based on convergent replicated data types (CRDTs). While Riak KV was built as a data-agnostic key/value store, Riak Data Types enable you to use Riak KV as a data-aware system and perform transactions on 5 CRDT-inspired data types:"
2. The Note can be removed here, as versions below 2.0 are EOL.